src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/page/customheaderlibs.html

```diff
@@ -18,6 +18,13 @@
      data-sly-test.clientLibRef="${symbol_dollar}{formstructparser.clientLibRefFromFormContainer}">
     <sly data-sly-test="${symbol_dollar}{clientLibRef}" data-sly-call="${symbol_dollar}{clientlib.css @ categories=clientLibRef}"/>
 </sly>
+#if (($includeForms == "y" || $includeFormsenrollment == "y" || $includeFormscommunications == "y") && $aemVersion != "cloud")
+<sly data-sly-use.clientlib="core/wcm/components/commons/v1/templates/clientlib.html"
+     data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
+     data-sly-test.themeClientLibRef="${symbol_dollar}{formstructparser.themeClientLibRefFromFormContainer}">
+    <sly data-sly-test="${symbol_dollar}{themeClientLibRef}" data-sly-call="${symbol_dollar}{clientlib.css @ categories=themeClientLibRef}"/>
+</sly>
```

Comment:
Why do we need this in archetype, it is already there in core-component repo?
cc @pankaj-parashar  @barshat7

Reply:
@barshat7 what is different script being added here as compared to archetype component core/fd/components/page/v1/page. What was the use case to provide a override at this level? In case of core component, the proxy component is not overriding.
cc : @rismehta